src/EditorFeatures/CSharpTest/SignatureHelp/AttributeSignatureHelpProviderTests.cs

```diff
@@ -356,6 +356,35 @@ class D
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        [WorkItem(12544, "https://github.com/dotnet/roslyn/issues/12544")]
+        public async Task TestAttributeWithOverriddenAndNewProperties()
+        {
+            var markup = @"
+class BaseAttribute : System.Attribute
+{
+    public virtual int foo { get; set; }
+    public int bar { get; set; }
+}
+class DerivedAttribute : BaseAttribute
+{
+    public override int foo { get; set; }
+    public new int bar { get; set; }
```

Comment:
Consider using a different type for the shadowed version so the test verifies we show the derived one, not the base.